docker/pre_commit_hook.sh

```diff
@@ -86,29 +86,15 @@ for arg in "$@"; do
     fi
 done
 
-# Check if dev-server is running and is healthy
-$(docker ps -a --format '{{json .}}' | grep $DEV_CONTAINER | jq .Status | grep -q healthy)
-is_container_running=$?
-
-if [ "$is_container_running" != "0" ]; then
-    # Start containers and run pre-commit hook
-    make start-devserver
-fi
-
 # Run hook in container
-CMD="$DOCKER_EXEC_COMMAND python3 ./$PYTHON_PRE_COMMIT_SYMLINK $@"
-echo "Running $CMD"
+GIT_USERNAME=$(git config user.name)
+GIT_USEREMAIL=$(git config user.email)
 
-$CMD
+docker compose run -T --no-deps --entrypoint "/bin/sh -c 'git config user.name $GIT_USERNAME && git config user.email $GIT_USEREMAIL && python3 ./.git/hooks/pre-commit-python $@'" dev-server
```

Comment:
add a comment explaining why we're doing this.

Reply:
done!